lib/acpi/acpi.c

```diff
@@ -749,6 +852,23 @@ ACPI_MADT_LOCAL_APIC *acpi_local_apic_get(int cpu_num)
 	return NULL;
 }
 
+int acpi_invoke_method(char *path, ACPI_OBJECT_LIST *arg_list, ACPI_OBJECT *ret_obj)
+{
+	ACPI_STATUS status;
+	ACPI_BUFFER ret_buff;
+
+	ret_buff.Length = sizeof(ACPI_OBJECT);
```

Comment:
minor: here also it would be better to use `sizeof(*ret_obj)` since that way there's no chance of using a different type than what `ret_obj` actually is.